src/js/utils/quality-labels.js

```diff
@@ -69,11 +69,25 @@ define([
         return Math.floor(bandwidth / 1000);
     }
 
+    // Use an empty object as the context and populate it like a hash map
```

Comment:
This comment applies to `return _.some...`, not to `hasRedundantLevels`. Please put the comment over the scope whose context you are referring to.